test/e2e/pkg/manifest.go

```diff
@@ -145,10 +145,10 @@ type Manifest struct {
 
 // ManifestNode represents a node in a testnet manifest.
 type ManifestNode struct {
-	// Mode specifies the type of node: "validator", "full", "light" or "seed".
+	// ModeStr specifies the type of node: "validator", "full", "light" or "seed".
 	// Defaults to "validator". Full nodes do not get a signing key (a dummy key
 	// is generated), and seed nodes run in seed mode with the PEX reactor enabled.
-	Mode string `toml:"mode"`
+	ModeStr string `toml:"mode"`
```

Comment:
What is the benefit of putting type into a variable name?

Reply:
We have `Testnet.Mode` and had `Manifest.Node`. They are from different types.

Ok, it is true that `type Mode = String` but they are different variables.

I renamed all variables in `Manifest` that collide with names in `Testnet`.

Reply:
I'm also trying to understand the motivation for these renames here, struct fields are scoped to the struct, so you can't experience 'collisions' in a sense that you can't access fields of one struct while referencing another, even if fields have the same names.

Could you please point in the code where this collision could happen ? If it's just a matter of providing context via the field name, I'd think we should just apply KISS principle and leave the way it is. But I might have missed the point with the rename.

Reply:
All good